src/VolleyManagement.Backend/VolleyManagement.UI/Areas/Mvc/ViewModels/Tournaments/TournamentTeamsListViewModel.cs

```diff
@@ -27,10 +27,14 @@ public TournamentTeamsListViewModel()
         /// </summary>
         /// <param name="source">All needed teams</param>
         /// <param name="tournamentId">Tournament id</param>
-        public TournamentTeamsListViewModel(List<TeamTournamentDto> source, int tournamentId)
+        public TournamentTeamsListViewModel(List<TeamTournamentDto> source, List<Division> divisions, int tournamentId)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Refactor this constructor to use a generic collection designed for inheritance. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=csharpsquid%3AS3956)